pep-0678.rst

```diff
@@ -3,7 +3,7 @@ Title: Enriching Exceptions with Notes
 Author: Zac Hatfield-Dodds <zac@zhd.dev>
 Sponsor: Irit Katriel
 Discussions-To: https://discuss.python.org/t/pep-678-enriching-exceptions-with-notes/13374
-Status: Accepted
+Status: Final
 Type: Standards Track
 Content-Type: text/x-rst
 Requires: 654
```

Comment:
Furthermore, add the canonical doc banner.

:warning: **IMPORTANT**: Unlike the other sections, a ref target label was not added for the relevant tutorial section, so it is impossible to reliably link it from elsewhere. I've opened a small CPython PR, python/cpython#106465 to add it; this PR should _not_ be merged before that, or the ref will be broken.

```rst
.. canonical-doc:: :meth:`BaseException.add_note` and :attr:`BaseException.__notes__`

   See :ref:`python:tut-exception-notes` for a user-focused tutorial.
```

Reply:
https://github.com/python/cpython/pull/106465 merged and backported, will take some time for a rebuild.